src/app/modals/createwallet/passphrase/passphrase.component.html

```diff
@@ -21,9 +21,9 @@
       <md-icon fontSet="ncIcon" fontIcon="nc-copy-2"></md-icon> Copy to Clipboard
     </button>
 
-    <button md-button *ngIf="!generate" class="paste TODO">
+    <button md-button *ngIf="!generate" class="paste TODO" (click)="pasteContent()">
```

Comment:
remove todo

Reply:
ok

Reply:
I have updated the PR description and shared the url of the task.